packages/cli/src/commands/buildHandler.js

```diff
@@ -134,6 +134,10 @@ export const handler = async ({
           // it could affect other things that run in parallel while building.
           // We don't have any parallel tasks right now, but someone might add
           // one in the future as a performance optimization.
+          //
+          // Disable the new warning in Vite v5 about the CJS build being deprecated
+          // so that users don't have to see it when this command is called with --verbose
+          process.env.VITE_CJS_IGNORE_WARNING = 'true'
```

Comment:
Vite 5 deprecated the CJS api. The only place the CJS api is used is in user's vite.config.ts files. We can disable the warning with https://github.com/vitejs/vite/blob/504bc5f80888ebd89d6c6726b18cb6f322ed538f/packages/vite/index.cjs#L29.